control-plane/grpc/src/operations/volume/traits_snapshots.rs

```diff
@@ -57,6 +59,32 @@ impl VolumeSnapshot {
     }
 }
 
+impl From<VolumeSnapshot> for models::VolumeSnapshot {
```

Comment:
move this to rest crate instead

Reply:
I get this error if I move this to rust crate (rest/service/src/v0/snapshots.rs):

```
174 | impl From<VolumeSnapshot> for models::VolumeSnapshot {
    | ^^^^^--------------------^^^^^----------------------
    | |    |                        |
    | |    |                        `rest_client::versions::v0::models::VolumeSnapshot` is not defined in the current crate
    | |    `grpc::operations::volume::traits::VolumeSnapshot` is not defined in the current crate
    | impl doesn't use only types from inside the current crate
```

Reply:
You won't be able to do so using impl directly on types, you'll have to add your own function or use newtype pattern.